tests/functional/base_pages/hub.page.js

```diff
@@ -33,12 +33,12 @@ class HubPage extends BasePage {
     return "h1";
   }
 
-  displayedGuidance() {
-    return '[data-qa="hub-guidance-panel"]';
+  displayedWarning() {
+    return '[data-qa="warning"]';
   }
 
-  displayedSubmissionGuidance() {
-    return '[data-qa="submission-guidance"]';
+  displayedGuidance() {
+    return '[data-qa="guidance"]';
```

Comment:
It's minor, and not introduced in this pr, but I can't see why we use "displayed". These should just be `warning` and `guidance`.

Reply:
I thought the same, but went with it as I thought there might be a reason. However if neither us knows, best to change it